### PR TITLE
Maxstep

### PR DESCRIPTION
**Pull request by maxsteep**

### CODE REVIEW

File: examples/freetype_misc.py [hunk 1]
```diff
@@ -68,58 +70,58 @@ def run():
...
-        screen, (480, 32), utext, colors["grey_light"], colors["red"], size=148
+        screen, (480, 32), utext, GREYLIGHTCOLOR, REDCOLOR, size=148
     )
 
     font.render_to(
         screen,
         (380, 380),
         "...yes, this is an SDL surface",
-        pg.Color(0, 0, 0),
+        (0, 0, 0),
```
Comment (rbsampang) [inline examples/freetype_misc.py]: I believe `(0, 0, 0)` is just `'black'` in colordict.py, so we could use that instead

File: src_py/colordict.py [hunk 2]
```diff
@@ -54,6 +54,7 @@
     "blue2": (0, 0, 238, 255),
     "blue3": (0, 0, 205, 255),
     "blue4": (0, 0, 139, 255),
+    "blue5": (50, 75, 245),
```
Comment (rbsampang) [inline src_py/colordict.py]: for the colors that we add into colordict, I think we should include the `alpha` value (4th value per color) to stay consistent with the other colors already there

**Review by rbsampang — COMMENTED** [review]
Nice work getting started on this! Left a few minor comments

File: src_py/colordict.py (hunk 2 above)
Comment (maxsteep) [reply src_py/colordict.py]: Completely agreed, thank you - excellent catch!

File: examples/freetype_misc.py (hunk 1 above)
Comment (maxsteep) [reply examples/freetype_misc.py]: For sure, completely agreed, I should have semantically linked that one too, thank you

**Review by rbsampang — APPROVED** [review]
Looks good!

**Pull request merged**